### ISSUE TITLE
bug: pip fails to solve requirement colossalai==0.1.9+torch1.11cu11.3

### ISSUE THREAD

**Issue by MEllis-github** [hpcaitech/OPT-Benchmark#6]
### Summary
Following https://github.com/hpcaitech/OPT-Benchmark#run-benchmarking, leads to the following error due to the requirement `pip install colossalai==0.1.9+torch1.11cu11.3 -f https://release.colossalai.org`.

```
ERROR: Could not find a version that satisfies the requirement colossalai==0.1.9+torch1.11cu11.3 (from versions: 0.1.8, 0.1.8+torch1.10cu10.2, 0.1.8+torch1.10cu11.1, 0.1.8+torch1.10cu11.3, 0.1.8+torch1.11cu10.2, 0.1.9, 0.1.9+torch1.12cu10.2, 0.1.9+torch1.12cu11.3, 0.1.10, 0.1.10+torch1.10cu10.2, 0.1.10+torch1.10cu11.1, 0.1.10+torch1.10cu11.3, 0.1.10+torch1.11cu10.2, 0.1.10+torch1.11cu11.3, 0.1.10+torch1.12cu10.2, 0.1.10+torch1.12cu11.1, 0.1.10+torch1.12cu11.3)
```

### Questions
* Which existing colossalai package/release is recommended and tested with OPT-Benchmarks?

### Files
* https://github.com/hpcaitech/OPT-Benchmark/blob/main/README.md

**Comment by MEllis-github** [comment]
Using image: https://github.com/orgs/hpcaitech/packages/container/pytorch-cuda/42358580?tag=1.12.0-11.3.0

**Comment by Cypher30** [comment]
I've solved this problem in the PR #10, thanks a lot for mentioning this problem, we had noticed that last week and currently working on it to provide you with modified version of test repo.
